### DISCUSSION TITLE
Mop Dock Mop Cleaning Frequency not recognized

### DISCUSSION THREAD

**Discussion by l0tzi** [Hypfer/Valetudo#1818] [Q&A / Support] [answered]
I am using the roborock s7 pro ultra. When I do a full clean up, the mop will be washed after each segment is finished, no matter what is configured in the Mop Dock Mop Cleaning Frequency setting.

**Comment by Hypfer** [comment, marked as answer]
Whoops. Indeed. I got something mixed up there when trying figure out what the correct parameters should be.
Also, my test segment just happens to take 10 minutes for a single pass which also happened to be the value I used for testing 🙃 

Fixed with
https://github.com/Hypfer/Valetudo/commit/15a6baaa8eef54a7c94f239559baba68e6efd59c
